caddyhttp/httpserver/plugin.go

```diff
@@ -449,6 +449,7 @@ var directives = []string{
 
 	// directives that add listener middleware to the stack
 	"proxyprotocol", // github.com/mastercactapus/caddy-proxyprotocol
+	"forwardproxy",  // github.com/caddyserver/forwardproxy
```

Comment:
This group is actually for plugins which implement [httpserver.ListenerMiddleware](https://godoc.org/github.com/mholt/caddy/caddyhttp/httpserver#ListenerMiddleware) which this does not (although I can see why you put it here) -- no big deal, just move it down to between `// directives that add middleware to the stack` and `"locale",` - so, still in the same place, but right now the grouping is just a little confusing. I should probably improve the wording of the comments too.